lrauv_ignition_plugins/test/test_mission_pitch_and_depth_mass_vbs.cc

```diff
@@ -83,14 +93,14 @@ TEST_F(LrauvTestFixture, PitchDepthVBS)
     EXPECT_GT(pose.Pos().Z(), -21.5);
 
     // Vehicle should exhibit minimal lateral translation.
-    EXPECT_NEAR(pose.Pos().X(), 0, 10); // FIXME(arjo): IMPORTANT!!
-    EXPECT_NEAR(pose.Pos().Y(), 0, 1e-1);
+    EXPECT_NEAR(pose.Pos().X(), 0, 1e-3);
```

Comment:
This tolerance may be a bit too tight. I think we can expect some amount of drift when running these missions especially since there is no control of the thruster.

Reply:
I was able to reduce it to `1e-5` in 92dcef0894521140d33c562458ea86c710173b13. Note that his corresponds to the previous Y, which had `1e-1`. I think sub-millimeter accuracy is pretty good here :100:

Reply:
I had to increase it to`2e-3` in c752c42a08082347b00157a11559b7efa57c256a because it was failing on CI. That's still less than the `1e-1` that we had before, but not as nice as `1e-5`...